code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -36,10 +36,12 @@
 /obj/item/circuit_component/bci
 	display_name = "Brain-Computer Interface"
 	display_desc = "Used to receive inputs for the brain-computer interface. User is presented with three buttons."
+	removable = FALSE
```

Comment:
I don't think this is actually used, just remove it. I was probably trying to do something else.